test/QueryBuilderTest.php

```diff
@@ -143,10 +209,10 @@ public function testGetSearchParamsWithAccessControl(string $queryString, string
         $this->assertSame($expected, $result);
     }
 
-    public function queryResultsWithAccessControl() : array
+    public function queryResultsWithAccessControl(): array
     {
         return [
-           'with user access control and role access control' => [
+            'with user access control and role access control' => [
```

Comment:
Space added, this was by code style linter?

Reply:
Yes.